source/extensions/filters/http/admission_control/BUILD

```diff
@@ -11,7 +12,7 @@ licenses(["notice"])  # Apache 2
 
 envoy_extension_package()
 
-envoy_cc_extension(
+envoy_cc_library(
```

Comment:
afaict this should be a `cc_library` rather than `cc_extension` - cc @htuch

Reply:
Seems right.